fhttp/http_client.go

```diff
@@ -277,6 +278,39 @@ func (h *HTTPOptions) AddAndValidateExtraHeader(hdr string) error {
 	return nil
 }
 
+func (h *HTTPOptions) ValidateConnectionReuseRange(inp string) error {
+	if inp == "" {
+		return nil
+	}
+
+	reuseRangeString := strings.Split(inp, ":")
+	var reuseRangeInt []int
+
+	if len(reuseRangeString) > 2 {
+		return fmt.Errorf("more than two integers were provided in the connection reuse range")
+	}
+
+	for _, input := range reuseRangeString {
+		if val, err := strconv.Atoi(input); err != nil {
+			return fmt.Errorf("invalid value for connection reuse range, err: %v", err)
```

Comment:
same, %w


btw can you see if one of the golang lint ci linter should be able to flag that? pretty sure I saw it mentioned by some linter before (outside of the linter that will say don't use fmt.Errorf at all that is, which I think is't right)

Reply:
I tried to find a linter that does that but didn't find anything useful. `go vet` can only flag when you don't provide enough argument to a format string but doesn't do type validation. Lemme know if you can find one.

Reply:
thx! I added the errorlint